### ISSUE TITLE
Option to shrink / compress the saved PNG file

### ISSUE THREAD

**Issue by GitMurf** [zsviczian/obsidian-excalidraw-plugin#486]
**Is your feature request related to a problem? Please describe.**
Often I have drawings / scratch notes that I don't need to keep the excalidraw file itself (can get quite large using apple pencil ink) and so I like to just save a "snapshot" of the drawing as PNG to have and reference in my notes. I do not need this to be super high resolution or anything and would rather have it be more like a screenshot (smaller file size).

**Describe the solution you'd like**
A setting to shrink the size/resolution/compress the drawings saved to PNG to make them smaller file size.

**Describe alternatives you've considered**
Currently I will take a screenshot on Windows with the snipping tool and just paste that into my vault instead because the file size is MUCH smaller.

**Comment by zsviczian** [comment]
Have you experimented with these settings? If you change the export scale to 1 the images won't be very large. Also, you can embed the png (or svg) into your document by default.

![image](https://user-images.githubusercontent.com/14358394/155898170-bdc1f1cb-26b8-4f7b-8e60-a6595b3691cc.png)
![image](https://user-images.githubusercontent.com/14358394/155898187-1e0d3d3c-0ef0-411f-9f8f-077816b62bfa.png)

**Comment by GitMurf** [comment]
Yep I have it set to 1 but drawings using ink are regularly over 1mb with the saved PNG. As opposed to if I just take a screenshot of it with the snipping tool on windows it will be 100kb or less usually.

**Comment by zsviczian** [comment]
I can add 0.5 to the scale. You can also test it yourself, just edit data.json and set pngExportScale to 0.5, and see what happens…
